server/app/services/seeding/DatabaseSeedTask.java

```diff
@@ -153,6 +167,21 @@ private Optional<QuestionDefinition> createQuestion(QuestionDefinition questionD
     }
   }
 
+  /** Seeds the predefined program categories from the category translation files. */
+  private void seedProgramCategories() {
+    CategoryTranslationFileParser parser = new CategoryTranslationFileParser(environment);
+    List<CategoryModel> categories = parser.createCategoryModelList();
+
+    categories.forEach(
+        category -> {
+          inSerializableTransaction(
+              () -> {
+                categoryRepository.fetchOrSaveUniqueCategory(category);
+              },
+              1);
+        });
+  }
```

Comment:
(blocking) Since you don't want a failure here to stop the server from loading, we probably want to wrap the whole thing in a try/catch as a last resort.

Reply:
Done.  I'm now catching any `RuntimeException` that may have slipped through the cracks.